src/regtech_mail_api/api.py

```diff
@@ -65,7 +65,14 @@ async def send_email(request: Request):
     sender_addr = request.user.email
     sender_name = request.user.name if request.user.name else ""
     type = request.headers["case-type"]
-    subject = f"[DEV BETA] SBL User Request for {type}"
+
+    header = "[BETA]"
+    if "cfpb" in sender_addr:
```

Comment:
```suggestion
    sender_addr.lower().endswith('cfpb.gov'):
```
...just to be extra safe and make sure we don't capture some `compliance-cfpb@somebank.com` type address?

Reply:
Good suggestion, will update.  There are some test CFPB domains we use to verify they’re not associated yet so we’ll just need to make sure they follow a *cfpb.gov format and not @cfpb-test.gov type thing.  Or building a configurable list of domains that will get flagged [CFPB BETA]

Reply:
Ok changed the check to split on the @ and check for "cfpb" in the domain place.  That will allow Bill's current test email accounts to stay the same format but would flag them appropriately.